python/katana/local/_graph.pxd

```diff
@@ -44,3 +44,6 @@ cdef class Graph(GraphBase):
 
     @staticmethod
     cdef Graph make(shared_ptr[_PropertyGraph] u)
+
+cdef class PyTxnContext:
```

Comment:
This should be called `TxnContext`. It's obviously python.

Also, how is this expected to be created and used? It's not clear how transactions are really created with this. Is there some `StartTransaction` operation?

Reply:
There is a set of python APIs for transactions, but was implemented and tested long time ago and is probably broken. I'll take a look at it, and see whether TxnContext can be fit into it, but in another PR.

Reply:
I think the API you are thinking of is in `katana.remote`. And it's not broken. It's tested in CI.

Please at least rename the class. And we need defaults so things keep working in the non-transactional mode until this stuff is completed.

Reply:
Set the "integrating transaction begin/commit/abort with `katana.remote`" a future task in Jira KAT-2570